packages/boinc/disable-whetneon.patch

```diff
@@ -1,30 +1,43 @@
-diff -uNr boinc-client_release-7.14-7.14.3/client/cs_benchmark.cpp boinc-client_release-7.14-7.14.3.mod/client/cs_benchmark.cpp
---- boinc-client_release-7.14-7.14.3/client/cs_benchmark.cpp	2020-03-10 03:47:48.000000000 +0200
-+++ boinc-client_release-7.14-7.14.3.mod/client/cs_benchmark.cpp	2020-03-20 14:06:28.788417500 +0200
-@@ -177,7 +177,7 @@
+diff -uNr boinc-client_release-7.16-7.16.11/client/cs_benchmark.cpp boinc-client_release-7.16-7.16.11.mod/client/cs_benchmark.cpp
+--- boinc-client_release-7.16-7.16.11/client/cs_benchmark.cpp   2020-08-30 16:04:59.000000000 +0800
++++ boinc-client_release-7.16-7.16.11.mod/client/cs_benchmark.cpp       2020-09-11 11:20:19.080000000 +0800
+@@ -176,7 +176,7 @@
+
      bdp->error_str[0] = '\0';
-     host_info.clear_host_info();
- 
+
 -#if defined(ANDROID) && defined(__arm__)
 +#if defined(ANDROID) && defined(__arm__) && !defined(__TERMUX__)
      // check for FP accelerator: VFP, Neon, or none;
      // run the appropriate version of Whetstone
      // (separated using namespaces)
-diff -uNr boinc-client_release-7.14-7.14.3/client/Makefile.am boinc-client_release-7.14-7.14.3.mod/client/Makefile.am
---- boinc-client_release-7.14-7.14.3/client/Makefile.am	2020-03-10 03:47:48.000000000 +0200
-+++ boinc-client_release-7.14-7.14.3.mod/client/Makefile.am	2020-03-20 14:01:33.838804535 +0200
-@@ -113,16 +113,6 @@
+diff -uNr boinc-client_release-7.16-7.16.11/client/Makefile.am boinc-client_release-7.16-7.16.11.mod/client/Makefile.am
+--- boinc-client_release-7.16-7.16.11/client/Makefile.am	2020-08-30 16:04:59.000000000 +0800
++++ boinc-client_release-7.16-7.16.11.mod/client/Makefile.am	2020-09-11 11:25:33.380000000 +0800
+@@ -1,8 +1,6 @@
+ ## -*- mode: makefile; tab-width: 4 -*-
+ ## $Id$
+ 
+-CXXFLAGS := $(filter-out -mfpu=vfpv3-d16,$(CXXFLAGS))
+-
+ include $(top_srcdir)/Makefile.incl
+ 
+ if ENABLE_CLIENT_RELEASE
+@@ -115,20 +113,6 @@
  
  boinc_clientdir = $(bindir)
  
 -if OS_ARM_LINUX
```

Comment:
This isn't a good way to disable this, no need to remove all the flags. Simply change this to another OS, like you did with `OS_DARWIN` above, and get rid of the prior way of patching by removing all flags, which is tougher to maintain.

Reply:
Yep, its a bit crazy to update this patch.
I think I can change this to a one liner by removing the detection, no guarantees though.
https://github.com/BOINC/boinc/blob/ebb0acebf97098336f89407258fbd89fc70f61bd/configure.ac#L1058